src/modules/esl-forms/esl-select-list/core/esl-select-list.less

```diff
@@ -8,8 +8,8 @@
   border: 1px solid @esl-select-list-secondary;
 
   min-height: 50px;
-  height: 400px;
-  max-height: 100%;
+  height: auto;
+  max-height: 50vh;
```

Comment:
This limitation basically sounds more like a dropdown one.
So possibly we can have some fixed limit here, and override for a list inside select-dropdown

Reply:
Yes, but if the list itself has few items ( I mean with no dropdown), there will be empty space just in the list. Is it ok, or I am missing sth?

Reply:
I think that is ok for a simple list.
But I'm just asking about fixed limitations like 400px for the max-height in case of the mostly "in-flow" element - list, while its usage in the dropdown will depend on the viewport height.